### ISSUE TITLE
PropertyTypeHintSniff does not consider parents

### ISSUE THREAD

**Issue by bendavies** [slevomat/coding-standard#904]
Given a Symfony command like:

```
class DeployCommand extends Command
{
    protected static $defaultName = 'deploy';
...
```

`PropertyTypeHintSniff` will complain:
```
 Property DeployCommand::$defaultName does not have native type hint nor @var annotation for its value.
```

adding ` /** @var string  */`
will result in:

```
Property DeployTagCommand::$defaultName does not have native type hint for its value but it should be possible to add it based on @var annotation "string".
```

I cannot add a native type:
```php
protected static string $defaultName = 'owsy:deploy-tag';
```

because the parent does not define a type
https://github.com/symfony/symfony/blob/0f46aa602a7a8a63fec1eee7aff59e439e3d5354/src/Symfony/Component/Console/Command/Command.php#L38
and would result in a compilation error
```
Type of DeployTagCommand::$defaultName must not be defined (as in class Symfony\Component\Console\Command\Command)
```

my only option is
```php
/**
 * @phpcsSuppress SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingNativeTypeHint
 * @var string
 */
```

**Comment by kukulich** [comment]
Yes, PHPCS works in one line scope. You can only use `@phpcsSuppress` or other ways how to disable sniff for one property.
